src/shogun/kernel/Chi2Kernel.cpp

```diff
@@ -15,7 +15,7 @@ using namespace shogun;
 void
 CChi2Kernel::init()
 {
-	SG_ADD(&width, "width", "Kernel width.", ParameterProperties::HYPER);
+	SG_ADD(&width, "kernel_width", "Kernel width.", ParameterProperties::HYPER);
```

Comment:
why this?

Reply:
Was getting and error that was saying it was double defined

Reply:
ok

Reply:
Actually, I have a hard time believing that there was no single test that ran the kernel constructor before ...

EDIT. I just checked, none of the base classes register a parameter called `width`, so why would this error appear?

Reply:
I am not sure. I will investigate.

Reply:
Changed it back and now it works with "width". Honestly, I don't know why I had an error before...

Reply:
lol well I guess nevermind then :)